src/auxsolvers/helmholtz_projector.cpp

```diff
@@ -42,7 +42,7 @@ HelmholtzProjector::Project(hephaestus::GridFunctions & gridfunctions,
     mfem::mfem_error(error_message.c_str());
   }
 
-  _h_curl_fe_space = fespaces.Get(_hcurl_fespace_name);
+  _h_curl_fe_space = fespaces.Get(_hcurl_fespace_name).get(); // NB: needs to be a pointer.
```

Comment:
Why does this need to be a raw pointer when `_h1_fe_space` does not?

Reply:
Because if there is no registered shared pointer, in the next couple of lines we set the variable equal to a pointer: __h_curl_fe_space = _div_free_src_gf->ParFESpace()_